src/layouts/MultiDhoLayout.vue

```diff
@@ -1,6 +1,29 @@
 <script>
 import { mapActions, mapGetters, mapMutations } from 'vuex'
 import BrowserIpfs from '~/ipfs/browser-ipfs.js'
+import gql from 'graphql-tag'
+import { timeago } from '~/utils/TimeUtils'
+import { parsedNotification } from '~/utils/notifications-utils'
+
+const NOTIFICATIONS_QUERY = `
+  queryNotification {
+    event{
+      eventType
+      name
+      id
+    }
+    user(filter: {eosAccountName: {eq: $account}}){
```

Comment:
@evgenibir this filter should be removed, since it wouldn't work with it ( if there are multiple users ). Also this filtering by eosAccountName will be done automatically when authentication is finished, so you don't need to filter it